components/script/dom/htmllinkelement.rs

```diff
@@ -377,6 +387,32 @@ impl HTMLLinkElementMethods for HTMLLinkElement {
     // https://html.spec.whatwg.org/multipage/#dom-link-target
     make_setter!(SetTarget, "target");
 
+    // https://html.spec.whatwg.org/multipage/#dom-link-crossorigin
```

Comment:
Using `enumerated_getter` always returns default crossorigin value as `anonymous`. This value is returned even if request is not a valid crossorigin request. This breaks many test case.